core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java

```diff
@@ -146,8 +147,9 @@ public final class VirtualHostBuilder implements TlsSetters {
     private ScheduledExecutorService blockingTaskExecutor;
     @Nullable
     private SuccessFunction successFunction;
+    @VisibleForTesting
     @Nullable
-    private Path multipartUploadsLocation;
+    Path multipartUploadsLocation;
```

Comment:
It is used [here](https://github.com/line/armeria/pull/4732/files#diff-2b0ddd858b27c417a1e3724dca38771ef2e2acc52f2e90eb2a7703ebcc7e4743R341-R344)

```java
assertThat(h2.multipartUploadsLocation()).isEqualTo(template.multipartUploadsLocation);
```

Reply:
Got it. Let me change its visibility and add getter for it. 🙆‍♂️

Reply:
I've added package-private getter for `multipartUploadsLocation` instead of using `@VisibleForTesting`

Reply:
@ikhoon Shouldn't we still need to add `@VisibleForTesting` to the getter?

Reply:
I think @ikhoon was referring to the field instead of the getter.   As you've said, I think the package-private getter with a `@VisibleForTesting` sounds good to me 👍

Reply:
> I think @ikhoon was referring to the field instead of the getter.

Right. This was my intention.